tine20/Tinebase/Server/WebDAV.php

```diff
@@ -37,6 +37,23 @@ public function handle(\Zend\Http\Request $request = null, $body = null)
             $this->_body = fopen('php://temp', 'r+');
             fwrite($this->_body, $request->getContent());
             rewind($this->_body);
+            /*
+             * JN: dirty hack for native Windows 7 & 10 webdav client (after early 2017):
+             * client sends empty request instead empty xml-sceleton -> inject it here
+             */
+            $broken_user_agent_preg = '/^Microsoft-WebDAV-MiniRedir\/[6,10]/';
```

Comment:
I haven't learned how, yet ;) Additionally I don't know how to test ist, do we have a proper DAV client?